.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
         entry: cargo fmt
         language: system
         types: [rust]
-        args: ["--all", "--", "--check"]
```

Comment:
Was this intentional?

Reply:
Yeah, I made it so that it formats the files instead of checking they're formatted. Because the schema file is auto generated, ie. not formatted automatically in the editor, this step would fail when just checking.